### ISSUE TITLE
How to change the section header or change tableview style

### ISSUE THREAD

**Issue by MarcSteven** [xmartlabs/XLForm#1082]
@mats-claassen  I reviewed all the docs about the framework, but cannot find a solution to change the section height .
It's so normal in the project, so does it support it ? And then I want to change the TableView style.

**Comment by mats-claassen** [comment]
Follow [this Readme section](https://github.com/xmartlabs/XLForm#how-to-customize-the-header-andor-footer-of-a-section) to change the header of a section.

XLForm uses a tableView so you can use all delegate methods of it

**Comment by MarcSteven** [comment]
@mats-claassen  So at the first time you should have a header view for section ,right?

**Comment by mats-claassen** [comment]
You can use that to set a header view for each section yes

**Comment by MarcSteven** [comment]
But if I want to make section header -same , how to do , for instance I have more rows -same layout but different data, so can I reuse the row? @mats-claassen

**Comment by mats-claassen** [comment]
I do not understand what you want to achieve but keep in mind that as XLForm uses UITableView underneath, you can achieve pretty much everything that you can with UITableView only (apart from row reuse). So if you want multiple section headers to be the same view but with different data, you can just do it using the functions linked above, just as you would do for an UITableView

**Comment by MarcSteven** [comment]
Yeah thank for your info, so perhaps should add doc or examples to help the people to use it. -add more features to do instead of UIOllectionview
